src/components/PageWizard/PageWizard.jsx

```diff
@@ -221,33 +108,37 @@ const PageWizard = ({
 
   return (
     <div
-      className={['page-wizard', className, hasStickyFooter ? 'page-wizard__sticky' : ''].join(' ')}
+      className={[
```

Comment:
this is a good use of the classnames library

Reply:
Good catch. Got to use classnames for the first time!